src/internal/operators/share.ts

```diff
@@ -176,7 +176,7 @@ export function share<T>(options: ShareConfig<T> = {}): MonoTypeOperatorFunction
       conn?.unsubscribe();
     };
 
-    return operate<T, T>((source, subscriber) => {
```

Comment:
Here before, we needed to make sure `lift` composed types, so we used a higher-order function. That is no longer necessary, so we can just inline things now.

That's why this one looks a little different.